### ISSUE TITLE
Bug when dealing with Double numbers from RDBs

### ISSUE THREAD

**Issue by julianrojas87** [RMLio/rmlmapper-java#206]
The condition used within [this function](https://github.com/RMLio/rmlmapper-java/blob/15c9c02c78aa743df02f7e773c2b1a229c3bc377/src/main/java/be/ugent/rml/access/RDBAccess.java#L351) to remove trailing decimal points `.0` is affecting proper double numbers. For example, `19.045` becomes `1945`.

The solution is simply checking if the number actually ends with `.0` to perform the transformation:
```java
if (DOUBLE.equals(dataType) && data.endsWith(".0")) {
   // remove trailing decimal points (Quirk from MySQL, see issue 203)
   return data.replace(".0", "");
}
```

**Comment by ghsnd** [comment]
Fixed on development branch

**Comment by DylanVanAssche** [comment]
Fixed in 950a77cb8f2a9254cf387cf3b6bbdca778fb49fb
